### ISSUE TITLE
_Columns slicing returns single _Column

### ISSUE THREAD

**Issue by JozsefKutas** [python-openxml/python-docx#770]
Slicing a _Columns instance returns a single _Column:

```python
import docx

document = docx.Document()
table = document.add_table(3, 3)
print(table.columns[1:]) // returns <docx.table._Colum n...>
```

This is similar to the behaviour for table rows raised in https://github.com/python-openxml/python-docx/issues/84. Slicing a _Rows instance now returns a list of _Row's as expected, but this behaviour doesn't seem to be implemented by _Columns.

I had a look in [table.py](https://github.com/python-openxml/python-docx/blob/master/docx/table.py) and it seems like _Columns and _Rows are implemented in quite different ways. This also leads to other inconsistencies, e.g. the error messages if indexes are out of bounds are different. Is there any reason for this?

My current workaround is to imitate the approach taken in _Rows:

```python
print(list(table.columns)[1:]) // returns [<docx.table._Column ...>, <docx.table._Column ...>]
```

**Comment by scanny** [comment]
No, no special reason. It would be better with slicing behavior.
